libbpfgo.go

```diff
@@ -328,6 +328,24 @@ func NewModuleFromFile(bpfObjPath string) (*Module, error) {
 	})
 }
 
+func SetStrictMode(mode string) error {
```

Comment:
Let's define a constant for this, similar to how we do for map and program types.